srcalc/src/test/java/gov/va/med/srcalc/web/view/VariableEntryTest.java

```diff
@@ -16,6 +19,30 @@
  */
 public class VariableEntryTest
 {
+    /**
+     * A map where the key is the variable key and the value is the ValueRetriever belonging
+     * to that variable.
+     */
+    private static final Map<String, ValueRetriever> LAB_MAP;
+    static {
```

Comment:
If you used `ImmutableMap`, you wouldn't need this static initializer block. They are a little obscure and best avoided.